presto-parser/src/main/java/com/facebook/presto/sql/tree/ExpressionTreeRewriter.java

```diff
@@ -855,6 +855,19 @@ protected Expression visitQuantifiedComparisonExpression(QuantifiedComparisonExp
 
             return node;
         }
+
+        @Override
+        public Expression visitGroupingOperation(GroupingOperation node, Context<C> context)
```

Comment:
add also visitor to `ExpressionExtractor`
and possibly
`DesugaringOptimizer` (noop since `GroupingOperation` doesn't occur in executable plan, should throw exception)